### PR TITLE
added support for day-of-week string

### LINKED ISSUE #1: Allow string day-of-week as well as numbers
`nthDay(3, 'friday', ...)` or `nthDay(3, 'fri', ...)` in addition to `nthDay(3, 5, ...)`

### PR DESCRIPTION
**Pull request by accraze**
the 2nd parameter can now  take either a number or string.

i.e. 

```
nthDay(3, 'friday', ...) or nthDay(3, 'fri', ...) in addition to nthDay(3, 5, ...)
```

resolves #1

### CODE REVIEW

File: src/nth-day.js [hunk 1]
```diff
@@ -25,6 +25,26 @@ function dateFromDate(date) {
   throw new Error('Do not know how to handle this type:', typeof date);
 }
 
+function dayFromString(dayOfWeek) {
+  if (typeof dayOfWeek === 'string'){
+    var dow = dayOfWeek.toLowerCase();
```
Comment (guyellis) [inline src/nth-day.js]: @accraze How about?

```
const dow = dayOfWeek.toLowerCase().slice(0, 3);
const weekDays = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
const dayNumber = weekDays.indexOf(dow);
if(dayNumber === -1) {
  throw...
}
return dayNumer;
```

**Comment by guyellis** [comment]
@accraze 1 comment - let me know what you think of the suggested change?

**Comment by guyellis** [comment]
I also just added a travis.yml file so that the Travis-CI tests should pass now.

**Comment by accraze** [comment]
@guyellis that suggested change is great! ... will fix that a little later today!

**Comment by radek** [comment]
:+1:

**Comment by guyellis** [comment]
@accraze Sorry it took so long to merge. I hadn't realized that you'd made the changes.

**Pull request merged**